src/components/Search/SearchOutputTable.vue

```diff
@@ -1,21 +1,18 @@
 <template>
-    <table class="table">
+    <table class="table table-hover">
         <thead>
             <tr>
-                <th>Name</th>
-                <th>Type</th>
-                <th>Subject</th>
-                <th>...</th>
+                <th v-for="(header, name, index) in headers" :key="index">
+                    {{name}}
+                </th>
             </tr>
         </thead>
-
-
         <tbody>
-            <tr>
-                <td><router-link to="/ABA">ABA Adult Mouse Brain</router-link></td>
-                <td>Standard</td>
-                <td>Neuroscience</td>
-                <td>...</td>
+        {{input_data}}
+            <tr v-for="(record, index) in input_data" :key="index">
+                <td v-for="(property, subIndex) in record" :key="subIndex">
+                    {{property}}
+                </td>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 16 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)